packages/extension/src/index.ts

```diff
@@ -47,13 +47,26 @@ const plugin: JupyterFrontEndPlugin<void> = {
       async execute(command: string, args: ReadonlyPartialJSONObject) {
         await commands.execute(command, args);
       },
-      listCommands() {
-        return commands.listCommands();
+      async listCommands() {
+        return await commands.listCommands();
       }
     };
 
     const endpoint = windowEndpoint(self.parent);
     expose(api, endpoint);
+
+    //TODO targetOrigin should be host page
+    window.parent?.postMessage('extension-loaded', '*');
+
+    // Add host css to Jupyter
+    window.onmessage = e => {
```

Comment:
This functionality of loading CSS could likely be provided by a separate JupyterLab command, which would then be discoverable by the host using the `jupyter-iframe-commands-host` package?